addons/mail/static/src/new/utils.js

```diff
@@ -291,3 +291,11 @@ export function useScrollSnapshot(refName, { onWillPatch: p_onWillPatch, onPatch
         });
     });
 }
+
+const { DateTime } = luxon;
+
+export function computeDelay(dateStr) {
```

Comment:
Would name this `computeDelayFromNow` that use a `computeDelay` function that accept 2 dateStr

Reply:
sure, once we need it. I think the idea is to merge PRs and adjust, so we don't have to manage many prs in the meantime until it's perfect (it never is anyway)